specification/iotoperations/IoTOperations.Management/models/broker/listeners.tsp

```diff
@@ -113,12 +110,12 @@ union TlsCertMethodMode {
 }
 
 @doc("Automatic TLS server certificate management with cert-manager")
-model AutomaticCertMethod {
+model CertManagerCertificateSpec {
   @doc("Lifetime of certificate. Must be specified using a Go time.Duration format (h|m|s). E.g. 240h for 240 hours and 45m for 45 minutes.")
   duration?: string;
 
   @doc("Secret for storing server certificate. Any existing data will be overwritten. This is a reference to the secret through an identifying name, not the secret itself.")
-  secretRef?: string;
+  secretName?: string;
```

Comment:
@jlian Shouuld this be `secretName` or `secretRef`

Reply:
99% sure it's `secretRef` https://github.com/Azure/iotedge-broker/blob/main/distrib/helm/MQ/examples/broker.yaml#L512

Reply:
As mentioned in chat, it actually should be `secretName` to match cert-manager. My mistake!